pom.xml

```diff
@@ -32,7 +32,7 @@
 		  If version cannot be resolved, make sure maven central repository is defined in settings.xml and the corresponding profile activated.
 		  If version in IntelliJ still cannot be resolved, a restart of IntelliJ might help!
 		-->
-		<version>2.0.3</version>
+		<version>2.1.1-SNAPSHOT</version>
```

Comment:
did you replace a release version with a snapshot version on purpose?

Reply:
To be discussed in the SDL weekly. For now this PR shows you the changes to the latest version.